web/src/js/ducks/_options_gen.ts

```diff
@@ -1,94 +1,94 @@
 /** Auto-generated by test_app.py:test_generate_options_js */
 export interface OptionsState {
-    add_upstream_certs_to_client_chain: boolean
-    allow_hosts: string[]
-    anticache: boolean
-    anticomp: boolean
-    block_global: boolean
-    block_list: string[]
-    block_private: boolean
-    body_size_limit: string | undefined
-    cert_passphrase: string | undefined
-    certs: string[]
-    ciphers_client: string | undefined
-    ciphers_server: string | undefined
-    client_certs: string | undefined
-    client_replay: string[]
-    client_replay_concurrency: number
-    command_history: boolean
-    confdir: string
-    connect_addr: string | undefined
-    connection_strategy: string
-    console_focus_follow: boolean
-    content_view_lines_cutoff: number
-    export_preserve_original_ip: boolean
-    http2: boolean
-    http2_ping_keepalive: number
-    http3: boolean
-    ignore_hosts: string[]
-    intercept: string | undefined
-    intercept_active: boolean
-    keep_host_header: boolean
-    key_size: number
-    listen_host: string
-    listen_port: number | undefined
-    map_local: string[]
-    map_remote: string[]
-    mode: string[]
-    modify_body: string[]
-    modify_headers: string[]
-    normalize_outbound_headers: boolean
-    onboarding: boolean
-    onboarding_host: string
-    proxy_debug: boolean
-    proxyauth: string | undefined
-    rawtcp: boolean
-    readfile_filter: string | undefined
-    rfile: string | undefined
-    save_stream_file: string | undefined
-    save_stream_filter: string | undefined
-    scripts: string[]
-    server: boolean
-    server_replay: string[]
-    server_replay_ignore_content: boolean
-    server_replay_ignore_host: boolean
-    server_replay_ignore_params: string[]
-    server_replay_ignore_payload_params: string[]
-    server_replay_ignore_port: boolean
-    server_replay_kill_extra: boolean
-    server_replay_nopop: boolean
-    server_replay_refresh: boolean
-    server_replay_use_headers: string[]
-    showhost: boolean
-    ssl_insecure: boolean
-    ssl_verify_upstream_trusted_ca: string | undefined
-    ssl_verify_upstream_trusted_confdir: string | undefined
-    stickyauth: string | undefined
-    stickycookie: string | undefined
-    stream_large_bodies: string | undefined
-    tcp_hosts: string[]
-    termlog_verbosity: string
-    tls_version_client_max: string
-    tls_version_client_min: string
-    tls_version_server_max: string
-    tls_version_server_min: string
-    udp_hosts: string[]
-    upstream_auth: string | undefined
-    upstream_cert: boolean
-    validate_inbound_headers: boolean
-    view_filter: string | undefined
-    view_order: string
-    view_order_reversed: boolean
-    web_columns: string[]
-    web_debug: boolean
-    web_host: string
-    web_open_browser: boolean
-    web_port: number
-    web_static_viewer: string | undefined
-    websocket: boolean
+    add_upstream_certs_to_client_chain: boolean;
```

Comment:
This here is a problem because that file is autogenerated when running the Python tests. We need to adjust the code that generates it so that they don't flipflop.

Reply:
> This here is a problem because that file is autogenerated when running the Python tests. We need to adjust the code that generates it so that they don't flipflop.
Weird question, but why would we source control an auto-generated file? 

I'll take a look at where this gets generated, but might take me some time as I'm still coming to grips with he repo.

Reply:
> Weird question, but why would we source control an auto-generated file?

Because this way we can run the web tests without pulling in and setting up an entire Python environment first.